src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,37 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name, health = 100) {
+    this.name = name;
+    this.health = health;
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name, health, hidden = false) {
```

Comment:
```suggestion
  constructor(name, health) {
```


no need to pass hidden as a parameter here - just add property hidden to the class
this.hidden = false;